packages/vyuh_node_flow/lib/graph/node_flow_events.dart

```diff
@@ -296,7 +297,7 @@ class ConnectionEvents<T> {
     ValueChanged<Connection>? onMouseLeave,
     void Function(Connection connection, Offset screenPosition)? onContextMenu,
     void Function(String nodeId, String portId, bool isOutput)? onConnectStart,
-    void Function(bool success)? onConnectEnd,
+    void Function(bool success, Offset position)? onConnectEnd,
```

Comment:
We should use the GraphPosition over here ... its an extension type that clearly tells the coordinate space here. We also have ScreenPosition that  is in the screen space.